test/functional/fixtures/regression/gh-987/test.js

```diff
@@ -0,0 +1,19 @@
+describe('[Regression](GH-987)', function () {
+    describe('Should show the target element fully while performing an action', function () {
```

Comment:
describe('Should fully show the target element on performing an action', function () {